Fixes some issues caused by split vocals

KarimAkra opened the pull request:
playing without split vocals was causing the game to resync vocals almost every frame for some reason so when your FPS drops a bit it could cause some sort of sound stuttering with the vocals, so this here fixes it
this also fixes player vocals volume still being 0 even after the opponent hit a note(while playing without split vocals ofc)

KarimAkra committed:
Update PlayState.hx

Trayfellow reviewed (approved):
nice, fixed the bug i had with week 6's vocals being stuttery after i paused and then resumed the song

KarimAkra committed:
remove that

the `!SONG.needsVoices` was making the inst and voices stop and not resume after pausing and resuming the game for some reason

cabfile commented:
does this fix the bug where the voices glitch if they're shorter than inst?

mcagabe19 commented:
no

cabfile commented:
is that ever going to be fixed

ShadowMario commented:
Does not fix the stuttering issue, as vocals are still being constantly resynced, so I'm closing this PR, feel free to make a new one if you find a definitive fix.
